packs/ansible/actions/lib/shell.py

```diff
@@ -17,9 +17,14 @@ def replace_args(attribute=None):
     def _replace_args(f):
         @functools.wraps(f)
         def _wrapper(self, *args):
+            def _replace(arg):
+                for rule in rules:
+                    if arg.startswith(rule):
```

Comment:
Won't `arg.startswith` mess these rules: 
```
'--verbose=vv': '-vv',
'--verbose=vvv': '-vvv',
'--verbose=vvvv': '-vvvv', 
```
when `--verbose=vvvv` in fact starts with `--verbose=vvv` and `--verbose=vv`?